common.lic

```diff
@@ -197,9 +197,14 @@ module DRC
   module_function :get_noun
 
   def empty_hands
```

Comment:
Yeah I can support that.

On Tue, Jan 12, 2016 at 2:34 PM, Robert Herbig <notifications@github.com>
wrote:

> In common.lic
> <https://github.com/rpherbig/dr-scripts/pull/174#discussion_r49509348>:
>
> > @@ -197,9 +197,14 @@ module DRC
> >    module_function :get_noun
> >
> >    def empty_hands
>
> Maybe just move this code there?
>
> I'm trying to reduce the chance someone uses the wrong helper.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rpherbig/dr-scripts/pull/174/files#r49509348>.
>